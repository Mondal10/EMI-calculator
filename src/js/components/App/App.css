body {
  background-color: #18af81;
}