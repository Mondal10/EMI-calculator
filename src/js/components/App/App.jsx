import React from 'react';

export const App = () => <div>Welcome to Medly Frontend Coding Challenge</div>;
