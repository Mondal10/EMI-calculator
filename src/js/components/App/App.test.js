import React from 'react';
import renderer from 'react-test-renderer';
import { App } from './App';

describe('App Snapshot', () => {
  it('Whole App rendered successfully', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
