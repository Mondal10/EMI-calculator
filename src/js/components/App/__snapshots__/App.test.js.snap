// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render successfully 1`] = `
<div>
  <div>
    Welcome to Medly Frontend Coding Challenge
  </div>
</div>
`;
